tools/bazar/fields/MapField.php

```diff
@@ -200,14 +219,9 @@ function geocodedmarkerRefresh( point )
         $GLOBALS['wiki']->AddJavascriptFile('tools/bazar/presentation/javascripts/geocoder.js');
 
         $geoCodingScript = '';
-        $latitude = '';
-        $longitude = '';
         if (isset($value)) {
-            $tab = explode('|', $value);
-            if (count($tab) > 1 && !empty($tab[0]) && !empty($tab[1])) {
-                $latitude = $tab[0];
-                $longitude = $tab[1];
-                $geoCodingScript .= 'var point = L.latLng('.$latitude.', '.$longitude.');
+            if (count($value) > 1 && !empty($value[$this->getLatitudeField()]) && !empty($value[$this->getLongitudeField()])) {
```

Comment:
```suggestion
            if (count($value) > 1) {
```

Je me demande si tous les tests sont nécessaires car getValue aura noramlement déjà testé ceci.
C'est juste une remarque.